scripts/mkdist.js

```diff
@@ -1,30 +1,30 @@
-var fs = require('fs')
+const fs = require('fs')
+const {join} = require('path')
+const mkdirp = require('mkdirp')
```

Comment:
> If you really want to drop the nodejs you will have to find someone else to write this as this means increasing the scope of this project quite large and loosing most attempts of having decent cross platform capabilites.

i don't intend to blow up scope at all, i want to keep it even smaller than the current PR. 

The mkdist changes aren't necessary here. I do intend to get rid of node as a requirement _in the long run_ (though not now and not asking you to). 

I also do not mean to make it harder to work with this repo than it needs to be-- which is why I made this using node the very first time: a quick + dirty solution that worked for me + david. And why i'm fine with ingesting the new node requirements, babel, and so on for the site. But mkdist does not need to ingest more dependencies.

I really appreciate you taking the time to clean other things up -- thank you! 

But the extra changes are beyond scope here -- as mentioned in last PR, all i asked for is changes to the _display_ of the site. (as an extreme, i would have expected the changes to be doable in css alone). Other changes happened, and that's very fine because it's what you prefer and the site looks way, way better now, etc. (thanks!!). But im not asking for mkdist and the rest of the repo to change.

---

I write the below to make it more clear _why_ i'm **against requiring installing node** in general  -- as much as I am against **requiring installing go or other large dev envs when not necessary**. I don't mean to turn this into a long discussion either -- only to express why i may be seemingly-unreasonably rejecting what _i think_ is the best package manager + smooth package install process created to date.

node + npm are certainly very nice, but you have to buy into installing node + npm which is _not_ that nice. then you have to keep them up to date, have to learn how to use new things, and debug them when they go wrong (which they inevitably do, often. even in npm deps get out of date and break). It's hard for regular node users to look back and realize just how hard it is to get going with node and installing things in the first place. it's not a "30 seconds and done forever" thing, it's something that introduces knowledge dependencies and forces the user to learn more things, inc some very complicated things.

Look, it's the same story with go. `go get` + compiling things is really, really nice. everything "just works", it's cross platform, everything is awesome. That is, _if_ you buy into installing go, setting up your env + path the right way, and you keep your packages nice and not breaking interfaces. By the time you get to installing the package, the user has had to learn a ton of things. (both Go and Node are similar in this, and why putting node/npm and golang people in the same room usually ends with people talking past each other about how their thing is the best without trying each other's stuff out. (fun for the reader: have rob pike and substack write programs in js and go respectively. hilarity ensues.)

Both node/npm and go models are really nice, if you buy into them fully, have them installed, and know how to use them. But the problem is it makes repos unapproachable to people outside of those circles, people who do not yet have these things installed, and who honestly should not need to. I've watched hundreds of people install Go and node just to use a small script or program. It's very painful.